infra/endpoints.tf

```diff
@@ -220,17 +216,16 @@ resource "aws_vpc_endpoint" "kinesis_stream_endpoint" {
 
 # TODO - remove and use the key we manage in this Terraform workspace
```

Comment:
I'm pretty sure this is unused and can be removed. The Lambdas in the dev account all seem to use either KID 16c7c492-6fe1-40f6-978e-d02cf5cbb966 or the default key for env var encryption.